recipe/meta.yaml

```diff
@@ -45,11 +46,23 @@ test:
     - test -f $PREFIX/lib/libva.so
     - test -f $PREFIX/lib/libva.so.2
     - test -f $PREFIX/lib/pkgconfig/libva.pc
-    - test -f $PREFIX/lib/pkgconfig/libva-drm.pc
```

Comment:
why is this file no longer needed, the `libva-drm.pc`

Reply:
I moved it to below.